packages/app/src/components/DBTraceWaterfallChart.tsx

```diff
@@ -81,7 +113,7 @@ function getConfig(source: TSource, traceId: string) {
       alias: 'Body',
     },
     {
-      valueExpression: alias.Timestamp,
+      valueExpression: sortKey,
```

Comment:
alias should be `Timestamp` ?

Reply:
ah yeah, it's the same thing but I will change it back to `alias.Timestamp`